src/storage/errors.rs

```diff
@@ -534,7 +534,7 @@ pub fn extract_key_error(err: &Error) -> kvrpcpb::KeyError {
             key_error.set_primary_mismatch(primary_mismatch);
         }
         _ => {
-            error!(?*err; "txn aborts");
+            warn!(?*err; "txn aborts");
```

Comment:
We may change it to `INFO` level if there is expected cases in this path.

In my understanding
- `ERROR` means something went wrong and the system may not work correctly
- `WARN` means there is unexpected problem happening, the system may continue to work

Reply:
I make it warn because this is for the `_` case: we don't know the actual error type here.